luarules/gadgets/unit_shield_behaviour.lua

```diff
@@ -158,8 +222,8 @@ local function triggerDowntime(unitID, weaponNum)
 	spSetUnitShieldRechargeDelay(unitID, weaponNum, 3600)
 
 	spSetUnitShieldState(unitID, weaponNum, false)
-	shieldData.downtimeReset = gameSeconds + shieldData.downtime
 	shieldData.shieldEnabled = false
+	spSetUnitRulesParam(unitID, shieldOnUnitRulesParamIndex, 0, {inlos = true})
```

Comment:
Perhaps do something like `local INLOS = {inlos = true}` at file scope so as not to create garbage tables each time.

Reply:
donezo